lte/gateway/release/build-ovs.sh

```diff
@@ -99,3 +102,6 @@ ksrc="/lib/modules/$kvers/build"
 sudo make -f debian/rules.modules KSRC="$ksrc" KVERS="$kvers" binary-modules
 
 cp /usr/src/*.deb ${WORK_DIR}
+
+# OVS packages:
+# sudo dpkg -i openvswitch-datapath-module-4.9.0-9-amd64_2.8.9-1_amd64.deb openvswitch-switch_2.8.9-1_amd64.deb openvswitch-test_2.8.9-1_all.deb python-openvswitch_2.8.9-1_all.deb  libopenvswitch_2.8.9-1_amd64.deb openvswitch-common_2.8.9-1_amd64.deb
```

Comment:
?

Reply:
I moved it to commit msg for list of packages to install.